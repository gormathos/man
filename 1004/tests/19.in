10
-6.59258 8.78005 -7.156 -1.04326 8.67346 9.57534 3.71749 -4.10448 -7.32405 -6.64762
10
-6.0736 3.72958 5.83496 0.902893 -9.44986 5.5106 9.24307 1.31357 -7.43076 3.59629
