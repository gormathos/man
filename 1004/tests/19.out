1.00000 2.12165 -231.88091 -778.07271 18825.89014 84357.12990 -570874.64571 -3279229.12686 2880299.66595 31405908.95318 26661667.53914
134046.62545 -850428.95531 -120036253.24797 54633168.77065 145210608.41528 -113620843.92037 -4545202.69063 64172668.83586 20238.61943 8351035.17679
587759.18295 -70461775.26091 -8715381.51492 27250741.85914 -313330689.90676 -30445782.87705 -6846218.91732 18479955.54573 -328071.70955 -67494817.98085
